Presentation/DualNavigationControllersSplitDelegate.swift

```diff
@@ -23,8 +40,14 @@ open class DualNavigationControllersSplitDelegate: NSObject, UISplitViewControll
 
     public let presentDetail = Delegate<UISplitViewController, Disposable>()
 
-    /// Returns a signal that will signal when collapsing or expanding
+    /// Returns a signal that will signal when collapsing or expanding with an initial value of true
+    @available(*, deprecated, message: "use `collapsedState` signal instead")
     public var isCollapsedSignal: ReadSignal<Bool> {
+        return isCollapsedProperty.readOnly().map { $0 ?? true }
+    }
+
+    /// Returns a signal that will signal when collapsing or expanding. Current value can be nil the collapsed state cannot be determined reliably yet.
+    public var collapsedState: ReadSignal<Bool?> {
```

Comment:
I'm fine with the `Bool?` as it states nicely what is going on (true, false or unknown), hence I doubt an enum will make things clearer (perhaps even the opposite). However, we could consider just naming this `isCollapsed` (as it is not taken and it is not that uncommon that we name read or readwrite signals without the signal postfix, especially when we don't have a name collision). If however you like to stay with the state variant, I would prefer `isCollapsedState` instead.